src/App.jsx

```diff
@@ -17,6 +17,7 @@ function App() {
         </a>
       </div>
       <h1>Vite + React</h1>
+      <p>Hello world</p>
```

Comment:
I don't see how this p element could be centered. Could you change the App functional component to contain only this p element, and add css to center it both ways?

Reply:
Ok